radio/src/functions.cpp

```diff
@@ -376,9 +376,7 @@ void evalFunctions()
           case FUNC_SET_TIMER:
             timerSet(CFN_TIMER_INDEX(cfn), CFN_PARAM(cfn));
             break;
-#endif
```

Comment:
Seemed redundant to me, and it would have been a 3rd repetition to also include the RANGECHECK/BIND case (below) which was causing warnings.  But either way would work.